kvirt/config.py

```diff
@@ -2662,7 +2662,10 @@ def create_openshift_iso(self, cluster, overrides={}, ignitionfile=None, podman=
                 templ = env.get_template(os.path.basename("ignition.j2"))
                 if hosts_content is not None:
                     hosts_content = base64.b64encode(hosts_content.encode()).decode("UTF-8")
-                ignition_url = f"http://{api_ip}:22624/config/{role}"
+                if jinjafilters.valid_ipaddress(api_ip) == "IPv4":
```

Comment:
can i suggest to check for presence of : in api_ip to decide whether it's an ipv6?
this way, we can use a oneliner and not introduce an extra filter

Reply:
sure, sounds good to me :)